HW_3/templates/register.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base.html" %}
+{% block body %}
+<h1>Register</h1>
```

Comment:
`flash` cообщениz, кстати, можно разместить тут.
```suggestion
<h1>Register</h1>
    {% with messages = get_flashed_messages() %}
    {% if messages %}
            <ul>
                {% for message in messages %}
                    <li>{{ message }}</li>
                {% endfor %}
            </ul>
        {% endif %}
    {% endwith %}
```